components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -150,13 +170,80 @@ function AutomaticPointsTable:queryPlacements(tournaments)
 			local tournamentIndex = tournamentIndices[result.tournament]
 			local tournament = tournaments[tournamentIndex]
 
-			result.points = tonumber(result.extradata.prizepoints)
+			result.prizePoints = tonumber(result.extradata.prizepoints)
 			result.securedPoints = tonumber(result.extradata.securedpoints)
+			result.extradata = nil
 			table.insert(tournament.placements, result)
+
+			local participant = result.participant
+			local teamIndex = reverseAliasLookupTable[tournamentIndex][participant]
+			if teamIndex ~= nil then
+				teams[teamIndex].results[tournamentIndex] = result
+			end
 		end
 	)
 
-	return tournaments
+	return teams, tournaments
+end
+
+function AutomaticPointsTable:getPointsData(teams, tournaments)

```

Comment:
Having a function longer than 35ish lines is almost never a good thing, so try extracting functions from this